src/main/java/com/github/technus/tectech/proxy/ClientProxy.java

```diff
@@ -33,8 +31,8 @@ public void registerRenderInfo() {
         QuantumGlassBlock.renderID = RenderingRegistry.getNextAvailableRenderId();
         RenderingRegistry.registerBlockHandler(QuantumGlassBlock.renderID, new QuantumGlassRender());
 
-        QuantumStuffBlock.renderID = RenderingRegistry.getNextAvailableRenderId();
-        RenderingRegistry.registerBlockHandler(QuantumStuffBlock.renderID, new QuantumStuffRender());
+        //        QuantumStuffBlock.renderID = RenderingRegistry.getNextAvailableRenderId();
```

Comment:
Why was this removed?

Reply:
It was completely unused clutter, so just ended up overwriting it with this stuff. We can always retrieve it from gits history if it is ever needed in future.